src/schema/v2/Alerts/index.ts

```diff
@@ -437,7 +418,6 @@ export const resolveAlertFromJSON = (alert) => {
 export const AlertsConnectionType = connectionWithCursorInfo({
   name: "Alert",
   nodeType: AlertType,
-  edgeFields: AlertsEdgeFields,
```

Comment:
ChatGPT has assured me that the removal of custom edge fields will just cause us to use defaulting graphql edge fields,
The connection type will still have edges, and each edge will have node and cursor, which is usually sufficient for most use cases.


Looking for ways to test this

Reply:
Yah, this is safe as this will just populate the edge area of the connection; nodes should still be returned. Buttt It might be worth running `yarn sync-schema:local` in volt against a running local MP and then `yarn relay` and `yarn type-check` just to see if any of this edge stuff is being used anywhere. Feels like this will break in consumers (unless _that_ code is already retired, in which case I am just rambling!)

Reply:
Good idea, my fear is that Force seems to be using this AlertsConnection for the `me { alertsConnection....}` BUT does not use these specific fields

Reply:
Cool yah, as long as things type-check we should be good 👍

Reply:
Thanks Chris!

I've run this MP schema change against local force
```
yarn sync-schema-local
yarn relay
yarn type-check
```

All looks good! Will merge after rebasing with latest `main`